.changeset/cyan-pants-move.md

```diff
@@ -0,0 +1,5 @@
+---
+"@google/generative-ai": minor
+---
+
+Add frequencePenalty and presencePenalty parameters support for generate content.
```

Comment:
This will show up as a release note in the public changelog so we try to do our best to format it for public consumption. You can use backticks to format code literals and symbols. Suggestion:
```
Add `frequencyPenalty` and `presencePenalty` parameters support for `generateContent()`.
```